package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.3.4",
+  "version": "0.0.0-semantically-released",
```

Comment:
It is just placeholder for humans.
We could even completely remove package version since it set before `npm publish` by `semantic-release pre`